packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -165,7 +166,10 @@ export const pivotQueryResults = ({
             ];
             const rowIndex = getByKey(rowIndices, rowKeys);
             const columnIndex = getByKey(columnIndices, columnKeys);
-            dataValues[rowIndex][columnIndex] = value;
+            dataValues[rowIndex][columnIndex] = {
```

Comment:
Does this make it significantly easier to render the conditional formatting? Since we can always determine the metric without this from the loop indices